router/router.go

```diff
@@ -13,6 +13,8 @@ func NewRouter(db database.Database) http.Handler {
 		userRoute.Post("/signup", SignUp(db))
 		userRoute.Post("/login", Login(db))
 		userRoute.Put("/passcode", UpdatePasscode(db))
+		userRoute.Get("/", FetchAllUsers(db))
+		userRoute.Delete("/", DeleteUser(db))
```

Comment:
There's no ID. How do we know what we're deleting?

Reply:
how do you mean? we have the user email and i am using that to delete the user details

Reply:
now i get what you mean.
there is a request body